src/graphql/ProductFragment.graphql

```diff
@@ -50,16 +50,21 @@ fragment ProductFragment on Product {
     {namespace: "taiga", key: "preOrderDateEstimate"},
     {namespace: "productListing", key: "preOrderStopCondition"},
     {namespace: "productListing", key: "launchType"},
+    {namespace: "productListing", key: "orderType"},
     {namespace: "productListing", key: "startingQuantity"},
     {namespace: "productListing", key: "endingQuantity"},
     {namespace: "productListing", key: "headline"},
     {namespace: "productListing", key: "sizing"},
     {namespace: "productListing", key: "campaignStart"},
-    {namespace: "productListing", key: "campaignDuration"},
+    {namespace: "productListing", key: "campaignEnd"},
     {namespace: "productListing", key: "productionDuration"},
```

Comment:
Shouldn't this be replaced with `productionStart`, as per the [Sept 19 revisions](https://docs.google.com/document/d/1Jn2NWuVh2BlokjsBIpk1IKg72bYAJDGPnR11dUqbNGc/edit?pli=1#heading=h.hgvtthxw8etr)? 

Or more specifically, based on [this thread](https://junipercreates.slack.com/archives/C05FA40TQ21/p1695159103637629), shouldn't we be doing this?
* `primaryProductionStart`
* `primaryProductionEnd`
* `secondaryProductionStart`
* `secondaryProductionEnd`

Reply:
Sorry, I hadn't updated the revisions yet, but based on my discussion with Nicholas which was reflected in this figma mock here - https://www.figma.com/file/zpHMXT2MCgsqA0rOIMVcsg/%F0%9F%8C%B8-Storefront?type=design&node-id=2063-248&mode=dev. 

The summary is:
1. We're keeping `productionDuration` and using `productionEnd` instead
2. `productionDuration` will be better since we can set a default to X days and with `productionEnd` set, whenever it's X many days before `productionEnd`, the timeline will set `production` to `inProgress`. 

The other more niche scenarios are highlighted in the Figma explanation, but we'll have some logic to handle when we're in between production end dates, and then finally when we're past both production end dates.

Reply:
This has been updated based on the conversation here - https://junipercreates.slack.com/archives/C05FA40TQ21/p1695331753218539.

Reply:
It has been further updated with `productionStarted` based on here - https://docs.google.com/document/d/1Jn2NWuVh2BlokjsBIpk1IKg72bYAJDGPnR11dUqbNGc/edit